fiasko_bro/validator_helpers.py

```diff
@@ -1,11 +1,40 @@
 from functools import wraps
 
 
-def tokenized_validator(token):
+def _general_tokenized_validator(tokens, check_method):
     def validator_decorator(func):
         @wraps(func)
         def func_wrapper(*args, **kwargs):
-            if token == kwargs.get('validator_token'):
-                return func(*args, **kwargs)
+            repo_tokens = kwargs.get('validator_token')
+            if repo_tokens is None:
+                repo_tokens = kwargs.get('validator_tokens')
```

Comment:
This can be rewritten as `repo_tokens = kwargs.get('validator_token') or kwargs.get('validator_tokens')`.